### ISSUE TITLE
test_tls_connect_revoked failed due to expired certificate

### ISSUE THREAD

**Issue by Ikke** [wbond/oscrypto#45]
`test_tls_connect_revoked` in `tests/test_tls.py` fails with

```
oscrypto.errors.TLSVerificationError: Server certificate verification failed - certificate expired 2020-04-13 12:00:00Z
```
Apparently the certificate for revoked.grc.com is expired.

**Comment by sethmlarson** [comment]
Might I suggest using [trustme](https://github.com/python-trio/trustme)?

**Comment by wbond** [comment]
I can make up a revoked cert without issue (I even run https://badtls.io for testing) - the point of this test is to use a real, in-the-wild revoked cert.

**Comment by wbond** [comment]
Looks like Digicert has some at https://www.digicert.com/digicert-root-certificates.htm

**Comment by wbond** [comment]
This is fixed by f85962b8f21d34923e6af0c71bc3a467bf6ad58e
